server/lib/forest.js

```diff
@@ -116,6 +116,8 @@ export default async function (app) {
           });
         });
         // Delete Collective
+        const newSlug = `${collective.slug}-${Date.now()}`;
+        await collective.update({ slug: newSlug });
```

Comment:
Thank you for doing that!

I forgot we had duplicated logic to delete collectives. Could we follow up and mutualize the code somewhere?